src/services/AccountManagementService.js

```diff
@@ -95,3 +95,7 @@ export const validateAndGenerateSeed = (seedPhrase, accounts) => {
         seedPhrase,
     };
 };
+
+export const importedSecretFound = (secret, accounts) => {
+    return accounts.find(x => x.keyPair.secret() === secret);
```

Comment:
nit picking: maby `return !!accounts.find(x => x.keyPair.secret() === secret)`, not necessary